styles/ShopUp.css

```diff
@@ -1,282 +1,335 @@
 :root {
-    --body-bg-light: #f1f5f9;
-    --body-bg: #e2e8f0;
-    --main-container-bg: #eff6ff;
-    --wrong-bg: #fee2e2;
-    --color-icon: #60a5fa;
+	--body-bg-light: #f1f5f9;
+	--body-bg: #e2e8f0;
+	--main-container-bg: #eff6ff;
+	--wrong-bg: #fee2e2;
+	--color-icon: #60a5fa;
 
-    --color-brand-light: #ffedd5;
-    --color-brand: #f97316;
+	--color-brand-light: #ffedd5;
+	--color-brand: #f97316;
 
-    --border-light: 2px solid var(--color-primary-light);
+	--border-light: 2px solid var(--color-primary-light);
 }
```

Comment:
Great Use Of CSS Variables
Also You can try numbering your variables like --gray-250

Reply:
Sure